Trench Warfare/map.xml

```diff
@@ -0,0 +1,181 @@
+            <helmet material="diamond helmet"/>
+            <chestplate material="iron chestplate"/>
+            <leggings material="iron leggings"/>
+            <boots material="iron boots"/>
+        </kit>
+    </class>
+</classes>
+<filters>
+    <team id="only-red">reds</team>
+    <team id="only-blue">blues</team>
+    <not id="no-tnt">
+        <material>tnt</material>
+    </not>
+    <not id="break-restrictions"> <!--Deny breaking stone brick and iron objects-->
+        <any>
+            <material>iron block</material>
+            <material damage="5">step</material>
+            <material>redstone wire</material>
+            <material>wall sign</material>
+            <material>stone plate</material>
+            <material>smooth brick</material>
+            <material>iron fence</material>
+            <material>smooth stairs</material>
+            <material>wood button</material>
+        </any>
+    </not>
+</filters>
+<regions>
+    <!--Playing Field Regions-->
+    <union id="bases">
+        <rectangle id="red-base" min="60,-17" max="76,18"/>
+        <rectangle id="blue-base" min="-61,18" max="-77,-17"/>
+    </union>
+    <union id="sides">
+        <rectangle id="red-side-right" min="59,-11" max="44,-17"/>
+        <rectangle id="red-side-left" min="59,12" max="44,18"/>
+        <rectangle id="blue-side-left" min="-60,12" max="-45,18"/>
+        <rectangle id="blue-side-right" min="-60,-11" max="-45,-17"/>
+    </union>
+    <union id="playable">
+        <rectangle id="main-area" min="59,-10" max="-60,11"/>
+        <region id="bases"/>
+        <region id="sides"/>
+    </union>
+    <union id="cannons">
+        <cuboid id="red-cannon1" min="68,75,-5" max="65,77,-5"/>
+        <cuboid id="red-cannon2" min="71,75,0" max="63,76,1"/>
+        <cuboid id="red-cannon3" min="68,75,6" max="65,77,6"/>
+        <cuboid id="blue-cannon1" min="-69,75,6" max="-66,77,6"/>
+        <cuboid id="blue-cannon2" min="-72,75,1" max="-64,76,0"/>
+        <cuboid id="blue-cannon3" min="-69,75,-5" max="-66,77,-5"/>
+    </union>
+    <apply block="never">
+        <negative id="playable-inverse">
+            <region id="playable"/>
+        </negative>
+    </apply>
+    <apply enter="only-red">
+        <rectangle id="red-home" min="60,-17" max="76,18"/>
+        <rectangle id="red-home-side-right" min="59,-11" max="44,-17"/>
+        <rectangle id="red-home-side-left" min="59,12" max="44,18"/>
+    </apply>
+    <apply enter="only-blue">
+        <rectangle id="blue-home" min="-61,18" max="-77,-17"/>
+        <rectangle id="blue-home-side-left" min="-60,12" max="-45,18"/>
+        <rectangle id="blue-home-side-right" min="-60,-11" max="-45,-17"/>
+    </apply>
+    <apply block="never">
+        <region id="blue-home"/>
+        <region id="red-home"/>
+    </apply>
+    <apply block-break="break-restrictions" block="no-tnt">
+        <complement>
+            <region id="playable"/>
+            <region id="red-home"/>
```

Comment:
What's the difference between `red-home` and `red-base`? It seems like you define the same region twice, or am I misreading? Same deal with blue.